### PR TITLE
Netstats deploy master

### PR DESCRIPTION
**Pull request by alastairong**

### CODE REVIEW

**Review by timotree3 — APPROVED** [review]
Blindly approving

**Review by zo-el — APPROVED** [review]

File: modules/default.nix [hunk 1]
```diff
@@ -32,9 +32,10 @@ with lib;
     ./services/membrane-proof-service.nix
     ./services/kv-uploader.nix
     ./services/lair-keystore.nix
-    ./services/match-service-api.nix
+    ./services/hp-stats-api.nix
+    ./services/mock-hpos-config.nix
```
Comment (zo-el) [inline modules/default.nix:36]: This looks like some residue of some old code

Comment (alastairong) [reply modules/default.nix:36]: They look correct to me?

Comment (zo-el) [reply modules/default.nix:36]: `mock-hpos-config` is not a service, this is from the old code that was supposed to help with the hpos-tests

`./services/mock-hpos-config.nix`

Comment (zo-el) [reply modules/default.nix:36]: Its also in the other two PR's that you have merged

**Pull request merged**